src/components/ErrorReport.vue

```diff
@@ -46,10 +46,7 @@
           <h4>Detailed Error Log</h4>
         </div>
         <div class="error-text">
-          <pre>
-         {{ errorLog }}
-        </pre
-          >
+          <pre v-html="errorLog"></pre>
```

Comment:
this is slightly risky. if anything in the error log looks like HTML it will get formatted, think like a script tag. Since you don't control the output this could be a way to send javascript to users.